Command/SwarrotCommand.php

```diff
@@ -59,13 +59,28 @@ protected function configure()
         ;
 
         if (array_key_exists('ack', $this->processorStack)) {
-            $this->addOption('requeue-on-error', 'r', InputOption::VALUE_NONE, 'Requeue in the same queue on error');
+            $this->addOption(
+                'requeue-on-error',
+                'r',
+                InputOption::VALUE_OPTIONAL,
```

Comment:
this does not make any sense given the naming of the option, and I'm quite sure it is broken